.github/workflows/main.yml

```diff
@@ -11,7 +12,6 @@ jobs:
       GDRIVE_FOLDER: ${{ vars.GDRIVE_FOLDER }}
     steps:
     - uses: actions/checkout@v3
-    - run: 'echo "hello"'
     - run: "pip install -r gdrive_requirements.txt"
     - run: "python test_pull_from_gdrive.py"
     - name: Archive pulled files
```

Comment:
GH interface won't let me comment on the line below. upload-artifact should be bumped to v3 there. v2 is resulting in a "Node12 is deprecated warning".

Reply:
changed to v3